lib/config/folderconfiguration.go

```diff
@@ -235,6 +241,14 @@ func (f *FolderConfiguration) prepare() {
 	if f.MarkerName == "" {
 		f.MarkerName = DefaultMarkerName
 	}
+
+	if runtime.GOOS == "android" && f.RawModTimeWindowS <= 0 {
+		f.cachedModTimeWindowS = 2
+	} else if f.RawModTimeWindowS < 0 {
+		f.cachedModTimeWindowS = 0
```

Comment:
I think this should be above android check, and we should still allow folks on android set it to -1 which means "don't touch it"

Reply:
This is always zero because of the fallthrough...

Reply:
Do we even need this branch? It will be zero by default, then android check can be folded into the switch.
I also think we should still check if its fat, as we pull in the lib and not use it now.